page-changelog.php

```diff
@@ -12,8 +12,217 @@
+<li>After switching to Postgresql changing any activity checkbox checks all of the boxes - <a href="https://github.com/owncloud/core/issues/23761">#23761</a></li>
+<li>Upgrade from o.C. 8.2.X to 9.0.X. Cannot access to oC if and LDAP user that made a share was deleted from the LDAP server. - <a href="https://github.com/owncloud/core/issues/24090">#24090</a></li>
+<li>Update sabre/dav to 3.0.9 for windows 10 support - <a href="https://github.com/owncloud/3rdparty/issues/269">3rdparty/#269</a></li>
+<li>Update sabre/dav to 3.0.9 on stable9 - <a href="https://github.com/owncloud/3rdparty/issues/270">3rdparty/#270</a></li>
+<li>Spinning wheel OC 9 - <a href="https://github.com/owncloud/activity/issues/481">activity/#481</a></li>
+<li>wrong links in RSS feed - <a href="https://github.com/owncloud/activity/issues/508">activity/#508</a></li>
+<li>LDAP users don't get listed when I try to share - <a href="https://github.com/owncloud/android/issues/1527">android/#1527</a></li>
+<li>Announcementcenter is listing only the last 5 announcements - <a href="https://github.com/owncloud/announcementcenter/issues/67">announcementcenter/#67</a></li>
+<li>[stable9] renameTag SQL bugfix - <a href="https://github.com/owncloud/bookmarks/issues/234">bookmarks/#234</a></li>
+<li>[stable9] move version to info.xml - <a href="https://github.com/owncloud/bookmarks/issues/238">bookmarks/#238</a></li>
+<li>work around filter_vars limitation with unicode chars in the URL - <a href="https://github.com/owncloud/bookmarks/issues/239">bookmarks/#239</a></li>
+<li>[9.0] Do not override user-provided title when bookmarking via bookmarklet - <a href="https://github.com/owncloud/bookmarks/issues/247">bookmarks/#247</a></li>
+<li>Adding bookmark leads to: SQLSTATE[22021]: Character not in repertoire: 7 ERROR: invalid byte sequence for encoding \"UTF8\" - <a href="https://github.com/owncloud/bookmarks/issues/230">bookmarks/#230</a></li>
+<li>Export Addressbook - <a href="https://github.com/owncloud/contacts/issues/248">contacts/#248</a></li>
+<li>Review dav urls as used in the docs - <a href="https://github.com/owncloud/documentation/issues/2074">documentation/#2074</a></li>
+<li>Acessing files via webdav with curl: wrong link - <a href="https://github.com/owncloud/documentation/issues/2330">documentation/#2330</a></li>
+<li>Remove non-existent warning for file locking - <a href="https://github.com/owncloud/documentation/issues/2351">documentation/#2351</a></li>
+<li>update php5-smbclient status - <a href="https://github.com/owncloud/documentation/issues/2362">documentation/#2362</a></li>
+<li>Re-added some performance hints. - <a href="https://github.com/owncloud/documentation/issues/2367">documentation/#2367</a></li>
+<li>Performance tuning: Recommend redis based file locking - <a href="https://github.com/owncloud/documentation/issues/2381">documentation/#2381</a></li>
+<li>Ubuntu 16.04 - <a href="https://github.com/owncloud/documentation/issues/2396">documentation/#2396</a></li>
```

Comment:
I took it out. @cmonteroluque can you filter this next time? I noticed but thought it was intentional...